src/bin/cargo/commands/vendor.rs

```diff
@@ -27,6 +27,12 @@ pub fn cli() -> App {
                 .help("Respect `[source]` config in `.cargo/config`")
                 .multiple(true),
         )
+        .arg(
+            Arg::with_name("explicit-version")
+                .short("-x")
+                .long("explicit-version")
+                .help("Always include version in subdir name"),
```

Comment:
Would you consider calling this something like `--version-dirs` or similar? "explicit" doesn't seem very self-explanatory when encountered outside the documentation.

Reply:
This is the same option that `cargo-vendor` had. I'm fine with renaming it though.

Reply:
Ah, I didn't realize that.

If you think there's value in providing a compatible option, you could provide this as an alias. I do think we should have a self-documenting option name though.

Reply:
OK, I'm inclined to drop the short parameter since `-x` wouldn't make sense, and it doesn't seem like a common enough feature to consume another letter for.